recipe/meta.yaml

```diff
@@ -10,32 +11,36 @@ package:
   version: {{ version }}
 
 source:
-  # we have to use git urls here to get the submodules needed for the
-  # build
-  git_url: https://github.com/dmlc/xgboost
-  git_tag: v{{ version }}
+  url: https://github.com/dmlc/xgboost/releases/download/v{{ version }}/xgboost-{{ version }}.tar.gz
+  sha256: 28bec8e821b1fefcea722d96add66024adba399063f723bc5c815f7af4a5f5e4
   patches:
     # xgboost patches
     - 0001-Force-endian-flag-in-cross-compilation-mode.patch  # [arm64 or aarch64 or ppc64le]
     - 0002-Enable-latest-libcxx-on-MacOS.patch  # [osx]
     - 0003-Use-mingw-w64-path.patch
     - 0004-Undo-dmlc-xgboost-9436.patch
+    - 0005-CMake-Explicitly-link-with-CCCL-standalone-or-CTK-10.patch
 
 build:
   number: {{ build_number }}
-  skip: True  # [cuda_compiler_version == "11.2"]
+  {% if python is defined %}
+  skip: {{ python.split(".")[:2] != min_python.split(".")[:2] }}
+  {% endif %}
```

Comment:
This checks that we only build Python packages once and that is for the minimum version that the packages support. It behaves similar to `noarch: python` that was added before. Though seems to be liked better by the linter